src/css/application.css

```diff
@@ -266,6 +266,23 @@ body {
   background-color: var(--color-gray);
 }
 
+.librarypicker__item.dashboard__menu-item_active::before {
+    content: '';
+    position: absolute;
+    border-color: #000;
+    border-style: solid;
+    border-width: 0 0.15rem 0.15rem 0;
+    height: .7rem;
+    left: 0.6em;
+    -webkit-transform: rotate(45deg);
```

Comment:
I believe [`autoprefixer`](https://github.com/postcss/autoprefixer#autoprefixer-) should take care of this